coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -149,6 +202,18 @@ def _parse_documentation_with_symbols(self,
                 retval_desc = line[return_offset:]
                 parsed.append(self.ReturnValue(desc=retval_desc))
 
+            elif ref_identifiers and \
+                    stripped_line.startswith(tuple(ref_starters)):
+                parse_mode = self.Reference
+                rawtypeRef = startswith(line, tuple(ref_starters))
+                typeRef = _extract_typeRef(ref_identifiers, rawtypeRef)
+                splitted = line.split(rawtypeRef, 1)[1].split(
+                    ref_identifiers[typeRef][1])
+                addr = splitted[0].strip()
+                desc = splitted[1].strip()
+                parsed.append(self.Reference(typeRef=typeRef,
+                                             ref_addr=addr, desc=desc))
+
             # These conditions will take care if the parsed section
             # descriptions are not on the same line as that of it's
```

Comment:
Also need to add a condition to check for `ref_addr` and `desc` in the next line!